### DISCUSSION TITLE
PESTPP-MOU with risk  - decision variables not in risk stack

### DISCUSSION THREAD

**Discussion by RyanConway91** [usgs/pestpp#277] [Q&A]
I am running PESTPP-MOU using a risk objective. I provided a parameter stack from a PESTPP-IES run. However, this ensemble does not have my MOU decision variables (which are the pumping rates of potential future pumping/injection wells). When I fire up PESTPP-MOU I get this:

![image](https://github.com/usgs/pestpp/assets/25184697/8c6c9f70-cf18-48cd-a498-6b9470af207a)

Obviously, _risk_ and my well rates were not in my PESTPP-IES calibration. 

I could do random uniform distribution sampling to get these parameters into my IES ensemble, but the risk and well rates will meaningless/wrong, right?

**Comment by jtwhite79** [comment]
I think the mention "forgive" at the bottom of your screen shot means mou should have kept going (and generated the initial population from a uniform distribution...).  Did it keep going?

**Reply by RyanConway91** [reply]
Yes, it kept going. It's not an issue that risk and the decvars are nonsense/random in the risk stack?

**Reply by jtwhite79** [reply]
Correct - mou is supposed to "forgive" an stack from ies that only has adj pars and not decvars...supposed to...

**Reply by RyanConway91** [reply]
Got it, thanks!
